pkg/cmd/start/start_test.go

```diff
@@ -289,25 +290,26 @@ func TestPipelineStart(t *testing.T) {
 			require.NotEmpty(t, lhjob.Spec.PipelineRunParams, "should have pipeline run params")
 		}
 
-		params := map[string]string{}
-
 		if name == "add-custom-envs" {
```

Comment:
I think it's cleaner to check that `tc.verifyEnvs` isn't nil

Reply:
Late but done via https://github.com/jenkins-x-plugins/jx-pipeline/pull/510/commits/3eae6fe424ddc4d2f4b7beffe7feff3e3ac77df1